docs/_themes/clusterhq/static/docs.css

```diff
@@ -0,0 +1,381 @@
+/* Docs */
+
+/* Nav Style from website */
```

Comment:
:disappointed: It is a shame to start a new site design on top of Bootstrap and use CSS instead of LESS.